dockers/docker-sflow/port_index_mapper.py

```diff
@@ -5,7 +5,7 @@
 import traceback
 from sonic_py_common.logger import Logger
 from socket import if_nametoindex
-from swsssdk import SonicV2Connector, port_util
+from sonic_py_common import port_util
```

Comment:
The port_util change split to another PR, this PR need wait for another PR merge first:
https://github.com/Azure/sonic-buildimage/pull/11347